config.json

```diff
@@ -6,5 +6,10 @@
     "REJECTED": "rejected",
     "DISPUTED": "disputed",
     "REVOKED": "revoked"
-  }
+  },
+  "PAGINATION_SIZE" : 10,
```

Comment:
move this to a secrets.json

add a sample secrets-sample.json

add secrets.json to gitignore.